### ISSUE TITLE
Mr Detective [Prompts]

### ISSUE THREAD

**Issue by TusharAMD** [TusharAMD/SuperSpeechSaga#22]
# Mr. Detective - Interrogate and Unravel Mysteries
![image](https://github.com/TusharAMD/SuperSpeechSaga/assets/59115865/e833f560-e012-4024-8179-896ed15963e6)

Welcome to Mr. Detective, a thrilling game where your interrogation skills are your greatest asset. Dive into complex mysteries, question suspects, and uncover the truth. This README will guide you through the essentials of our game, providing you with the knowledge to get started and become the master detective you were always meant to be.

Please Note: This issue is for prompts only. You are expected to create suitable prompts that will generate dialogs.

## How to begin

1. Fork and clone this repo to local machine
2. cd into the game directory 
3. Make your changes
4. Commit to a new branch and make pull request

**Comment by Ar7109** [comment]
@TusharAMD, Assign this to me.

**Comment by Gaurav-576** [comment]
I'd like to work on this issue and get to know more about it's documentation and about the issue.

**Comment by TusharAMD** [comment]
@Ar7109 assigned the project.
@Gaurav-576 Hi Gaurav, since ankur was first to comment I assigned issue to them. But you are free to create such similar issues on issues tab. About the things you are supposed to do in the "prompts" issues is that you are supposed to create prompts that would generate dialog.
For eg. I can tell gemini 
![image](https://github.com/TusharAMD/SuperSpeechSaga/assets/59115865/4cdaf6fb-41aa-41d3-bae4-6989d8b146f2)

The green marked part is prompt. You can make 30-40 prompts like this that can be used in game. Make sure the prompts are related to each other and follows storyline.

**Comment by Gaurav-576** [comment]
I see, are there any other game characters then @TusharAMD ? I'd like to generate their dialogs in that case.

**Comment by TusharAMD** [comment]
@Gaurav-576 Umm right now there is no storyline. I was thinking that person who creates prompts would create a story too. I think I should create a separate issue to get story and based on that other contributors can make game assets, dialogs and actual game.

**Comment by Gaurav-576** [comment]
Yeah that would be really great. Once the story is ready it would be easy for contributors to create prompts for the story.

**Comment by Ar7109** [comment]
@TusharAMD, Ok I got it, I will make the prompts. 
Thanks for assigning.

**Comment by ananyabansal16** [comment]
If this issue is still pending, I can take this up. Lemme know. :)
